.travis.yml

```diff
@@ -16,5 +18,8 @@ matrix:
   include:
     - env: BUILD_SYSTEM=qmake
     - env: BUILD_SYSTEM=cmake
+    - env: BUILD_VARIANT=click
```

Comment:
Better just `PLATFORM=click`

Reply:
Possible values for PLATFORM could be `linux-desktop`, `click`, later maybe also `osx`, `android`, `ios`. Instead of `click` also `ut` or `ubuntu-touch` or `linux-mobile-click` or `linux-click-arm` or what ever could be used (just pick one).